rpm_spec/subpackages/manageiq-gemset

```diff
@@ -59,4 +59,24 @@ Requires: network-scripts
 %{gemset_root}/specifications
 %{gemset_root}/vmdb
 %{gemset_root}/enable
+%{_prefix}/lib/systemd/system/manageiq-amazon*
```

Comment:
Does it matter what we name the unit files?  wondering if we should just prefix all of these with manageiq-providers-*, then it's a one-liner

Reply:
Also, can we have a subdirectory?  `.../lib/systemd/system/manageiq/

Reply:
https://github.com/ManageIQ/manageiq-rpm_build/pull/134#discussion_r566930587

> Unfortunately you can't have a subdirectory like /lib/systemd/system/manageiq as far as I know, we'd have to install them somewhere and symlink them all over.

Reply:
> Does it matter what we name the unit files? wondering if we should just prefix all of these with manageiq-providers-*, then it's a one-liner

I suppose not, as long as we split out core and plugins into their own rpms which we are already doing here by looking at the gemset

Reply:
Actually no sorry, this is saying which files in the target directory are part of this rpm so if we said `%{_prefix}/lib/systemd/system/*` that would include the core workers as well.

Reply:
If all plugin files will have `manageiq-providers` prefix, manageiq-core can be written as:

```
%{_prefix}/lib/systemd/system/manageiq*
%exclude %{_prefix}/lib/systemd/system/manageiq-providers*
```

Reply:
Thanks @simaishi looks much better, updated